.github/workflows/fuzzydata-test.yml

```
@@ -0,0 +1,33 @@
+name: master
+on:
+  push: 
```

Can we test this in the PR first, then change this after we know it works?
```suggestion
  pull_request
```

---

Done!